modules/@apostrophecms/ui/ui/apos/components/AposButton.vue

```diff
@@ -424,6 +434,10 @@ export default {
   }
 }
 
+.apos-button--icon.apos-button--small {
+  padding: 7.5px 10px;
```

Comment:
7.5? Does the half px really do anything?

Reply:
don't be mean, we're already using this padding set